openrpc.json

```diff
@@ -22,19 +22,19 @@
               "newPendingTransactions",
               "syncing"
             ],
-            "description": "The type of subscription to create. Must be one of:`newHeads`: new block headers - `logs`: logs matching a filter object - `newPendingTransactions`: new pending transactions - `syncing`: changes in syncing status"
+            "description": "The type of subscription to create. Must be one of the following: 1) `newHeads` - New block headers. 2) `logs` - Logs matching a filter object. 3) `newPendingTransactions` - New pending transactions. 4) `syncing` - Changes in syncing status."
           }
         },
         {
           "name": "filterOptions",
           "schema": {
             "title": "filterOptions",
             "type": "object",
-            "description": "An optional object containing filter options specific to the subscription type. Only applicable for 'logs' subscription type.",
+            "description": "An object containing filter options specific to the subscription type. Only applicable for the `logs` subscription type.",
             "properties": {
               "address": {
                 "type": "string",
-                "description": "A single address or an array of addresses to filter the logs by. Optional."
+                "description": "(Optional) A single address or an array of addresses to filter the logs by."
```

Comment:
is the other option `topics` required? we can `required` in the schema to better describe it